Server/routes/Mobile/items.js

```diff
@@ -26,4 +26,21 @@ router.get('/:articleId', async (req, res) => {
   }
 })
 
+router.get('/:articleId/similar', async (req, res) => {
```

Comment:
not looking for authorization ?

Reply:
no, because the user should be able to see the articles without being logged in